lib/generators/decidim/templates/app.json.erb

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "<%= app_name %>",
+  "description": "TODO: Add a short description about <%= app_name %>",
+  "keywords": [
+    "small",
+    "sharp",
+    "tool"
+  ],
+  "addons": [],
```

Comment:
Postgres is detected automatically and about the other ones I'm not really sure.. that's a bit personal, isn't it?